### ISSUE TITLE
Missing env var when running `reaction` via standalone NPM script i.e. outside Docker

### ISSUE THREAD

**Issue by bayareacoder** [reactioncommerce/reaction-platform#134]
I want to run the GraphQL API outside Docker by executing `npm start:dev` in `reaction` folder after installing npm modules.
Error shown:
 Missing environment variables:
    STRIPE_API_KEY: A private Stripe API key
although this key is set in the .env file.

I inspected the API plugins and see in the Stripe plugin that this key is required and is filled out with a default value but only when in test mode. 
I changed `NODE_ENV` from `development` to `test` and then indeed there is no error.

So it seems you are not reading in env vars and so this cannot run outside Docker (although doc says it is possible but 'not recommended') ?
### Versions
ReactionCommerce 3.7.1
Node 12.18.0
NPM 6.14.4

**Comment by ticean** [comment]
Hi @bayareacoder We have a large community who runs Reaction on a variety of systems. We've settled on Docker to normalize the development environment and provide a consistent runtime. Docker's consistency helps us support for everyone.

You're welcome to run Reaction outside of Docker but you'll need to bring your own batteries. We've only included them in the standard workflow.

The example configurations are in each project's `.env.example`. Use any means you'd like to set those in the application environment before starting it and the app will use them and behave as expected.
